README.md

```diff
@@ -49,24 +55,23 @@ the [REST API](https://github.com/ehrbase/webtester/wiki/REST-API-Reference).
 
 - Java 11
 - Apache Maven
-- Apache JMeter (optional)
 
 ### Build the application
 
 ```shell
 $ mvn clean install
 ```
 
-The previous command requires:
+## Data Loading
 
-- Apache JMeter installed on your computer.
-- `JMETER_HOME` environment variable defined.
+The WebTester can also fill the DB with dummy data. For this you need to configure. 
```

Comment:
It may be helpful to mention that you have to call the endpoint and with which parameters